main.go

```diff
@@ -311,6 +315,7 @@ func (cfg *config) runcmd(command, pattern string, files ...string) error {
 func cmdNew(c *cli.Context) error {
 	cfg := c.App.Metadata["config"].(*config)
 
+	var title string
 	var file string
 	if c.Args().Present() {
 		file = c.Args().First()
```

Comment:
it may not needed, because file is inferred from title at https://github.com/mattn/memo/pull/3/files/6fd66d51c6f35092edc77d34e1f8b06efc85964a#diff-7ddfb3e035b42cd70649cc33393fe32cR333